app/models/user.rb

```diff
@@ -4,7 +4,7 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
-  has_many :tournaments
+  has_many :tournaments, dependent: :destroy
```

Comment:
Similarly not sure we want to destroy tournaments when a user is destroyed - just in case someone deletes their account It feels weird to delete the tournament as well (in case it was accidental during the tournament)